sdk/core/core/src/az_http_pipeline.c

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <az_http_pipeline.h>
+#include <az_http_policy.h>
+
+#include <_az_cfg.h>
+
+az_http_policy pipeline[8] = {
+  { .pfnc_process = az_http_pipeline_policy_uniquerequestid, .data = NULL },
+  { .pfnc_process = az_http_pipeline_policy_retry, .data = NULL },
+  { .pfnc_process = az_http_pipeline_policy_authentication, .data = NULL },
+  { .pfnc_process = az_http_pipeline_policy_logging, .data = NULL },
+  { .pfnc_process = az_http_pipeline_policy_bufferresponse, .data = NULL },
+  { .pfnc_process = az_http_pipeline_policy_distributedtracing, .data = NULL },
+  { .pfnc_process = az_http_pipeline_policy_transport, .data = NULL },
+  NULL,
+};
+
+AZ_NODISCARD az_result
+az_http_pipeline_process(az_http_request_builder * const hrb, az_mut_span const * const response) {
+  AZ_CONTRACT_ARG_NOT_NULL(hrb);
+  AZ_CONTRACT_ARG_NOT_NULL(response);
+
+  return pipeline[0].pfnc_process(&(pipeline[1]), hrb, response);
```

Comment:
can you return here:
```
return az_http_pipeline_nextpolicy(&pipeline[0], hrb, response);
```
???
Think it's a little more easy to understand what this is doing and it would be reusing what all other policies do

Reply:
`&(pipeline[1])` => `pipeline + 1`